page-search.php

```diff
@@ -35,9 +35,11 @@
 				<?php endif; ?>
 			</div>
 
-			<div class="content-main flex-container">
+			<div id="content" class="content has-sidebar">
 				<?php get_template_part( 'content', 'pagefull' ); ?>
+				<?php get_sidebar(); ?>
 			</div>
 		</div>
 		<?php endwhile; // end of the loop. ?>
```

Comment:
I think this is the line that's causing the 500 error. It looks like the beginning of the while loop was removed (was around 28, right after the inc/breadcrumbs call)